src/components/forums/Forum.js

```diff
@@ -190,6 +223,60 @@ const Forum = () => {
         return body;
     }
 
+    // API - Edit your own comments
+        const handleEditComment= (selectedComment) => {
+        if (comment === '') {
+            setCommentError(true);
+            setCommentErrorText('Please enter your comment');
+            return false;
+        } else {
+            //console.log("about to call api edit forum");
+            callApiEditComment(selectedComment);
+            setShowEditAlertMessage(true);
+            setTimeout(() => {
+                window.location.reload();
+            }, 100);
+        }
+    }
+    const callApiEditComment = async () => {
+        const url = `${REACT_APP_API_ENDPOINT}/editForumComment`;
+        console.log(url);
+
+        const response = await fetch(url, {
+            method: "POST",
```

Comment:
you need to update the API to a `PUT` method. Here is where you would change the method being used.

Reply:
Does this change anything else? I have never used a "PUT" method.

Reply:
no, `PUT` methods operate similarly to `POST` methods in terms of taking in a body as a request